### PR TITLE
Clarify some aspect of choosing plugin groups/IDs

### PR DESCRIPTION
**Pull request by jbartok**

### CODE REVIEW

**Comment by jbartok** [comment]
@bot-gradle

**Comment by jbartok** [comment]
@bot-gradle test this

File: subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc [hunk 1]
```diff
@@ -77,22 +77,31 @@ include::sample[dir="snippets/developingPlugins/publishingPlugins/kotlin",files=
...
 This is done in a `plugins` block within the `gradlePlugin` block.
-The most important part is the `id` property, as that both uniquely identifies it on the {portal} and prevents namespace clashes between different plugin authors.
+The most important part is the `id` property, as that both uniquely identifies it on the {portal} and prevents namespace clashes between different
+plugin authors.
 
-As a convention, we recommend you use an ID based on the reverse-domain pattern used for Java packages, for example `org.example.greeting`.
-You can use this example ID for the Greeting Plugin project, but substitute the values for more appropriate ones if you're working with your own plugin that you actually want published.
+If you would like to associate your plugin with a particular organization, you also set the ID based on that organization’s domain using the
+reverse-domain pattern used for Java packages, for example `org.example.greeting`. If the plugin doesn't belong to any specific organization,
+then the plugin ID should be associated with the author, for example by using the authors GitHub ID in a reverse domain pattern, like `io.github.johndoe`.
```
Comment (tresat) [inline subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc:85]: author's

File: subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc [hunk 2]
```diff
@@ -77,22 +77,31 @@ include::sample[dir="snippets/developingPlugins/publishingPlugins/kotlin",files=
...
 <2> Set the version for this publication. You need to increase the version, if you already published the plugin before.
-<3> Each plugin in a bundle is specified in the `plugins` blocks. As you are only publishing a single plugin at this point there will only be one entry, but should your project publish a bundle in the future you will list each of them in here.
+<3> Each plugin in a bundle is specified in the `plugins` blocks. As you are only publishing a single plugin at this point there will only be
+    one entry, but should your project publish a bundle in the future you will list each of them in here.
 <4> The name for each plugin block does not affect the plugin configuration, but needs to be unique for each plugin provided.
 <5> Set the unique `id` of the plugin.
 <6> Set the plugin name in human-readable form.
-<7> Set a description that will be displayed on the portal. Although this is optional, it provides useful information to people would might want to use your plugin. Please give careful thought to the value of this property.
+<7> Set a description that will be displayed on the portal. Although this is optional, it provides useful information to people would might
```
Comment (tresat) [inline subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc:103]: Is this optional?  I thought we were requiring these.

**Review by tresat — APPROVED** [review]

File: subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc (hunk 2 above)
Comment (ljacomet) [reply subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc:103]: Indeed, as we are editing this, let's remove the optional part, it really is not.

**Comment by jbartok** [comment]
@bot-gradle test and merge

**Pull request merged**